api/v1alpha3/azurecluster_default_test.go

```diff
@@ -396,7 +446,7 @@ func TestSubnetDefaults(t *testing.T) {
 								Name:          "cluster-test-controlplane-subnet",
 								CIDRBlocks:    []string{DefaultControlPlaneSubnetCIDR},
 								SecurityGroup: SecurityGroup{Name: "cluster-test-controlplane-nsg"},
-								RouteTable:    RouteTable{Name: "cluster-test-node-routetable"},
+								RouteTable:    RouteTable{},
 							},
```

Comment:
@CecileRobertMichon this is what you expected? I tested it and seems to work

Reply:
yep that's what I had in mind